rf_api_client/models/users_api_models.py

```diff
@@ -10,10 +10,20 @@ class UserDto(ApiBaseModel):
     name: Optional[str]
     surname: Optional[str]
     avatar: Optional[str]
-    birthday: Optional[datetime]  # todo check
+    birthday: Optional[datetime]
     is_extension_user: Optional[bool]
+    language: Optional[str]  # en-US
+    timezone: Optional[str]  # Europe/Moscow
```

Comment:
last three are not optional

https://srv.nppsatek.ru:42917/rf/rf-frontend/-/blob/dev/src/api/user.ts#L69

https://srv.nppsatek.ru:42917/pavlovma/sup/-/blob/dev/service/service/models/User.py#L37
https://srv.nppsatek.ru:42917/pavlovma/sup/-/blob/dev/service/service/core/users.py#L47